### PR TITLE
added to geographic_coordinate to pixel and pixel to geographic_coord…

### PR DESCRIPTION
**Pull request by basaks**
Added to refpixel geographic_coordinate to pixel tests
Added to refpixel pixel to geographic_coordinate tests

### CODE REVIEW

File: tests/test_refpixel.py [hunk 1]
```diff
@@ -471,6 +471,7 @@ def x_y_pixel():
     return itertools.product(x, y)  # returns a matrix of 5x5 random x, y pairs
 
 
+@pytest.mark.skipif(PYTHON3P6, reason='Skipped in python3p6')
```
[the remark this answers is not in the document]
Comment (basaks) [reply tests/test_refpixel.py:474]: the reason is due to us using a modern subprocess `run` function in the `test` only which does not have a feature we are using in the `test` only. I.e., there is no impact in python3p6 to run pyrate code.

**Review by mcgarth — APPROVED** [review]
Thanks for the change. The proposed two tests add valuable unit coverage

**Review by tfuhrmann — APPROVED** [review]
I've checked the change in the code of test_refpixel.py and checked the results using 'pytest'. Works as expected. I'm wondering though if the +1/-1 buffer is needed for the test data sets. In my test runs the gdallocationinfo result and the result of the convert_pixel_value_to_geographic_coordinate function were always identical for the 25 sets of test coordinates, e.g.:
gdal output:
```
  Location: (13P,36L)
  Band 1:
    Value: 284
```
convert_pixel_value_to_geographic_coordinate function result:
`13 36`

I remember a discussion with Sheece on this where we found differences of one pixel in the two calculation methods. But it might be that this was related to legacy code.

**Comment by tfuhrmann** [comment]
> I've checked the change in the code of test_refpixel.py and checked the results using 'pytest'. Works as expected. I'm wondering though if the +1/-1 buffer is needed for the test data sets. In my test runs the gdallocationinfo result and the result of the convert_pixel_value_to_geographic_coordinate function were always identical for the 25 sets of test coordinates, e.g.:
> gdal output:
> 
> ```
>   Location: (13P,36L)
>   Band 1:
>     Value: 284
> ```
> 
> convert_pixel_value_to_geographic_coordinate function result:
> `13 36`
> 
> I remember a discussion with Sheece on this where we found differences of one pixel in the two calculation methods. But it might be that this was related to legacy code.

Alright, I've tried the code without the +/-1 and re-run the test 100 times. Turns out that there is some locations where the +/-1 is needed, e.g.:
```
>           assert f"({x}P,{y}L" in out
E           AssertionError: assert '(27P,6L' in 'Report:\n  Location: (26P,6L)\n  Band 1:\n    Value: 290\n'
```

So, all good with the current implementation!

**Pull request merged**